test/controllers/requests_test.rb

```diff
@@ -1,14 +1,30 @@
 require 'test_helper'
 
 class RequestsTest < ActionDispatch::IntegrationTest
+  include ActiveSupport::Testing::MethodCallAssertions
```

Comment:
I'd rather we don't introduce mocking if possible. What I'm thinking is we should be able to do something like:

```ruby
assert_nil Peek.adapter.get(request.env['action_dispatch.request_id'])
```

To determine if the request did or didn't save any data.